packages/php/email-editor/src/Engine/PersonalizationTags/class-personalization-tags-registry.php

```diff
@@ -23,6 +23,7 @@ class Personalization_Tags_Registry {
 
 	/**
 	 * Initialize the personalization tags registry.
+	 * This method should be called only once.
```

Comment:
Not for this Pull request, but should we somehow ensure that the function is called only once? After it is called, set some property and then prevent calling it again.